crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -4,8 +4,9 @@
  <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    {# Asking search engines not to index our site while we're in development mode. #}
+    <meta name="robots" content="noindex">
     <title>U.S. Department of Justice - Civil Rights Division</title>
-    <!-- TODO: properly manage static assets -->
```

Comment:
@LindsayYoung My assumption is that this was a TODO from before we had static assets up and running and is now safe to remove; is that right?

Reply:
yes